frontend/packages/dev-console/src/components/topology/filters/TopologyFilterBar.tsx

```diff
@@ -71,15 +72,53 @@ const TopologyFilterBar: React.FC<TopologyFilterBarProps> = ({
   const kialiLink = getNamespaceDashboardKialiLink(consoleLinks, namespace);
   const queryParams = useQueryParams();
   const searchQuery = queryParams.get('searchQuery') || '';
+  const [isQuickSearchActive, setIsQuickSearchActive] = React.useState<boolean>(false);
 
   const onTextFilterChange = (text) => {
     const query = text?.trim();
     onSearchChange(query);
   };
 
+  React.useEffect(() => {
```

Comment:
You can probably re-use `useDocumentListener` for adding event listeners for keyboard shortcut.

Reply:
Yeah I saw that but its not a generic hook and is supposed to be used when we want some components to be visible only
when the user is actively interacting with the document.